### PR TITLE
DEVDOCS-3806: [update] Get transaction orders endpoint

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-3806](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3806)

## What changed?
* Remove `giftcertificate` and `storecredit` values from gateway
* Add `bigcommerce` as a gateway value
* Remove `status` field from gift_certificate object

[DEVDOCS-3806]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-3806?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/orders.v3.yml [hunk 1]
```diff
@@ -1863,7 +1864,7 @@ components:
                 - paypalexpress
                 - paypalpaymentsprous
                 - paypalpaymentsprouk
-                - plugnpay
+                - plugnplay
```
Comment (slsriehl) [inline reference/orders.v3.yml:1867]: AFAIK service is called `Plug'n Pay`, so I think this should remain `plugnpay`.

```suggestion
                - plugnpay
```

**Comment by slsriehl** [comment]
@bc-tgomez the linting errors seem pretty straightforward. I suggest resolving the easier ones, then coordinating with @markcmurphy to decide whether this should wait until we update the source file to OAS3.

**Comment by slsriehl** [comment]
@bc-tgomez I updated the tree to match the description given the current main branch. Do these changes still need to be made?

**Comment by bc-tgomez** [comment]
Migrated ticket to the new [Repo](https://github.com/bigcommerce/docs/pull/28).

**Pull request closed without merging**